events/open-standup.md

```diff
@@ -78,7 +78,7 @@ _Cross-dependencies / Requests for Pairing_
 
 -
 
-_New Milestones / Repos / Blog posts_
+_New Milestones / Repos / Blog posts (prompt each TL for team's milestone: Auction, Gallery, Platform, Grow, Discover, Purchase)_
```

Comment:
I would maybe phrase this as New Milestones / Repos / Blog Posts / New features or updated functionality released: prompt Auction, Gallery, Platform, Grow, Discover, Purchase teams for those In this proposed version seems like this prompt in () is part of Blog Posts.